src/FsAutoComplete/LspServers/AdaptiveFSharpLspServer.fs

```diff
@@ -2735,8 +2735,7 @@ type AdaptiveFSharpLspServer(workspaceLoader: IWorkspaceLoader, lspClient: FShar
               [| if config.LineLens.Enabled <> "replaceCodeLens" then
                    if config.CodeLenses.Signature.Enabled then
                      yield! decls |> Array.collect (getCodeLensInformation p.TextDocument.Uri "signature")
-                 // we have two options here because we're deprecating the EnableReferenceCodeLens one (namespacing, etc)
-                 if config.EnableReferenceCodeLens || config.CodeLenses.References.Enabled then
+                 if config.CodeLenses.References.Enabled then
```

Comment:
This mentions deprecating `EnableReferenceCodeLens`.  Should I remove that setting entirely or is there issues with removing settings like this?

Reply:
That seems fine - we can actually set a `deprecationMessage` on the config value that it's no longer in use in Ionide.